README.md

```diff
@@ -1,3 +1,4 @@
-This is the repo of Karen Avner and Tal Bar Nahor's capstone project!
-The name of our project is "food-mood", 
-and it's a web application that helps you decide where to order food from!
+# Heading 1
```

Comment:
I think the heading should have some content :)

Reply:
hahaha ok that was a funny misunderstanding on my side. fixed